server.js

```
@@ -1,30 +1,30 @@
-//const authJwt = require("./config/authJwt");
-const express = require("express");
 const db = require("./models");
+const express = require("express");
 const cors = require("cors");
-const http = require("http");
 require("dotenv").config();
 const bodyParser = require("body-parser");
 const cookieParser = require("cookie-parser");
-const fs = require("fs");
 const app = express();
+const path = require("path");
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)